### DISCUSSION TITLE
Connect with ArcGIS Pro?

### DISCUSSION THREAD

**Discussion by adamekcerv** [supabase/supabase#9059] [General]
Hi,
I would like to connect my supabase database with postgis extension to ArcGIS Pro and be able to edit data using it.
Is it somehow possible?
Thank you.

**Comment by GaryAustin1** [comment]
Does it need more than a database connection?
https://supabase.com/docs/guides/database/connecting-to-postgres#direct-connections

**Reply by adamekcerv** [reply]
The problem is that I am not able to connect to my default "postgres" supabase database on ArcGIS. I was able to create a new database using PgAdmin, and then the connection worked. But all the data were read only and the database disappeared in PgAdmin.
